### PR TITLE
added unit tests

### PR DESCRIPTION
**Pull request by DiegoDiazGlobant**
Added unit tests.

Preview:
tests_preview
cov

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -32,10 +32,12 @@ android {
...
 
 dependencies {
-
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'com.google.android.material:material:1.2.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+
+    testImplementation 'junit:junit:4.13.1'
+    testImplementation 'org.mockito:mockito-core:3.3.3'
     testImplementation 'junit:junit:4.+'
```
Comment (SartiMau) [inline app/build.gradle]: check that you have this dependency repeted

File: app/src/test/java/com/example/calculator/mvp/presenter/PresenterTest.java [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+import static com.example.calculator.utils.StringUtilsTest.ZERO;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class PresenterTest {
+
+    private CalculatorModel model;
+    private CalculatorView view;
+    private CalculatorPresenter presenter;
```
Comment (SartiMau) [inline app/src/test/java/com/example/calculator/mvp/presenter/PresenterTest.java]: remember to define these variables from the contract

**Review by BQuinteros — APPROVED** [review]

**Review by SartiMau — CHANGES_REQUESTED** [review]

**Review by BenjaAbdalaGlobant — APPROVED** [review]

File: app/build.gradle (hunk 1 above)
Comment (DiegoDiazGlobant) [reply app/build.gradle]: Solved!

File: app/src/test/java/com/example/calculator/mvp/presenter/PresenterTest.java (hunk 2 above)
Comment (DiegoDiazGlobant) [reply app/src/test/java/com/example/calculator/mvp/presenter/PresenterTest.java]: Yes! Solved.

File: app/src/test/java/com/example/calculator/mvp/presenter/PresenterTest.java [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+        model.setNewOperator(PRODUCT);
+        model.setNewOperator(SUB);
+        model.setNewOperand(THREE);
+        presenter.onEqualsButtonPressed();
+        assertEquals(DOUBLE_FIFTEEN, model.getResultValue());
+        verify(view).showResultValue(model.getResultValue());
+    }
+
+    @Test
+    public void EqualsButtonPressedWithDivOperationAndNegativeNumbersTest() {
```
Comment (SartiMau) [inline app/src/test/java/com/example/calculator/mvp/presenter/PresenterTest.java]: all methods tests start with a lowe case just like all methods

**Review by SartiMau — CHANGES_REQUESTED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**